pom.xml

```diff
@@ -1671,6 +1663,12 @@
                 <version>8.5.13</version>
             </dependency>
 
+            <dependency>
```

Comment:
We should add this to Airbase

Reply:
Agree. I'll cleanup after airlift is released and this PR is merged

Reply:
https://github.com/airlift/airbase/pull/397